src/controllers/users_controller.js

```diff
@@ -1,108 +1,120 @@
-const async = require("async");
-const usersModel = require("../models/users_model");
+const async = require('async');
+const usersModel = require('../models/users_model');
 
 module.exports.setUp = () => {
-    usersModel.setup();
+  usersModel.setup();
 };
 
 module.exports.findAllUsers = (req, res) => {
-    usersModel.findAllUsers((err, users) => {
-        res.status(err ? 500 : 200).json(err ? undefined : users);
-    });
+  usersModel.findAllUsers((err, users) => {
+    res.status(err ? 500 : 200).json(err ? undefined : users);
+  });
 };
 
-module.exports.register = ({body: {name = null, email = null, password = null}}, res) => {
-    if (!email || !password || !name) {
-        return res.status(400).json({
-            error: "Email, name and Password must be provided"
-        });
-    }
+module.exports.register = ({ body: { name = null, email = null, password = null } }, res) => {
+  if (!email || !password || !name) {
+    res.status(400).json({
+      error: 'Email, name and Password must be provided',
+    });
+  } else {
```

Comment:
Why are we changing to this approach, or is this just a lint thing

Reply:
this is the standard from the airbnb-base